### ISSUE TITLE
Add Surround (multi-channel) support to Phazor

### ISSUE THREAD

**Issue by JensDA** [Taiko2k/TauonMusicBox#764]
Playing multi-channel files used to work with the GStreamer backend.
Unfortunately, Phazor does not seem to support it, outputting those files in stereo instead.
Please consider supporting surround sound again.

**Comment by Taiko2k** [comment]
Yeah unfortunately that will have been lost with the removal of the GStreamer backend. Implementing this in Phazor, while I don't have immediate plans to do this, it could be a consideration.

So I can get an idea of the scope, what formats do you use for these multi-channel tracks? And how many channels do they use?

**Comment by JensDA** [comment]
I'm using flac with mostly 6 (5.1) channels. I have a few files with 8 (7.1) channels as well.
